Sentence_trie.py

```diff
@@ -11,6 +13,7 @@ def __init__(self, word, father, children=None):
         logging.info('Node created with word: %s', self.word)
 
 
+@timeit
```

Comment:
Timeit decorator is for debug purposes. Prints the execution tine of the function.